examples/jsm/tsl/display/BloomNode.js

```diff
@@ -364,7 +364,7 @@ class BloomNode extends TempNode {
 
 		// gaussian blur materials
 
-		const kernelSizeArray = [ 3, 5, 7, 9, 11 ];
+		const kernelSizeArray = [ 6, 10, 14, 18, 22 ];
```

Comment:
Can we add a comment here to explain the modification? Something like:

> To avoid a blocky blur, the gaussian coefficients are based on custom kernel radii. That required to double the original kernel sizes to retain the bloom strength, see #31528 for more details.